ethereum/core/src/test/java/org/hyperledger/besu/ethereum/forkid/ForkIdTest.java

```diff
@@ -667,18 +743,276 @@ public static Collection<Object[]> data() {
             "check13ByzantiumWithRemoteAnnouncingRinkebyPetersburg",
             Network.network(
                 GenesisHash.MAINNET,
-                asList(1150000L, 1920000L, 2463000L, 2675000L, 4370000L, 7280000L)),
+                asList(1150000L, 1920000L, 2463000L, 2675000L, 4370000L, 7280000L),
+                emptyList()),
             7987396L,
+            0L,
             empty(),
             empty(),
             wantPeerCheck("0xafec6b27", 0L, false)
-          }
+          },
+
+          // Fork ID test cases with block number and timestamp based forks
+          // Withdrawals test cases
+          {
+            "Withdrawals // Unsynced",
```

Comment:
Withdrawals test cases worthy of its own test class?

Reply:
Doesn't seem like quite the right split. But this test class is getting quite large. Maybe should split the test data out into test files? Although I suggest another PR to do that.